src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -592,12 +593,17 @@ export default class AircraftModel {
         let assignedAltitude = this.mcp.altitude;
         let flightPlanAltitude = this.fms.flightPlanAltitude;
 
+
         if (assignedAltitude === INVALID_NUMBER) {
             assignedAltitude = '-';
+        } else {
+            assignedAltitude *= UNIT_CONVERSION_CONSTANTS.FT_FL;
```

Comment:
this should be moved up to the original assignment on ln 593

```
let assignedAltitude = this.mcp.altitude * UNIT_CONVERSION_CONSTANTS.FT_FL;
```

same with `flightPlanAltitude`. That way we can get rid of the `else`s